plugins/modules/dcnm_network.py

```diff
@@ -1975,6 +1975,133 @@ def failure(self, resp):
 
         self.module.fail_json(msg=res)
 
+    def dcnm_update_network_information(self, want, have, cfg):
+
+        if cfg.get("vrf_name", None) is None:
```

Comment:
We need not compare mandatory objects here because they have to be included. Just check

Reply:
Network name is the only mandatory parameter.